docs/src/pages/company/software-engineer-intern/software-engineer-intern.md

```diff
@@ -0,0 +1,78 @@
+- **Contribute to deep, meaningful refactors and feature releases**. Material-UI is a complex codebase. Components we've shipped recently, such as the Tree View, Autocomplete, and Slider have required weeks of dedicated, careful work.
+- **Collaborate with the community**. Many small as well as meaningful fixes and features have been contributed by the community. Your role as a core team maintainer is to draw the best out of the community — to inspire those across the world to create and contribute through your reviews of their issues and pull requests.
+- **Experiment and play**. Great, unexpected features and heisenbug fixes have come from a number of sources — relentlessly methodical processes of elimination, free-flowing team collaboration, inspiration by adjacent libraries and projects, and difficult-to-explain individual strokes of brilliance. Whatever your preferred style is for creating new things that others might not have thought of, you'll find a welcome home on the team.
+
+### The best parts of this job
+
+- **You'll be at the cutting edge of application development** — working on one of the fastest-growing UI libraries on the market.
+- **You'll be part of an active, open, friendly community** of developers that are really excited about building awesome applications.
+- **Your work will be visible** as being part of one of the most popular open source project.
+
+### The worst parts of this job
+
+- **Shifting context.** You will necessarily have to shift context and dive into a different feature before the current one is done. It may even be in an area of the code base you're unfamiliar with or don't have a ton of understanding about. It's fun, rewarding work, but it can be very challenging.
+- **We move quickly, but don't sacrifice quality**. We ship early, often, and quickly. You may not be initially comfortable with the cadence with which we ship high-quality features and improvements to end-users. By doing so, we sacrifice on solving each problem 100% in exchange for fast feedback. Solving 50-70% of the issue with quality should be enough for any given iteration. Our users quickly tell us when we haven't pushed a solution far enough.
+- **Material-UI is a large codebase**. You may bang your head against the wall at times, and then write tests to assist future you 😌.
+  The work you will be doing is somewhat unique and idiosyncratic. You probably have not had a similar role before.
+- **In open-source, you're faced with a nonstop stream of bug reports and support requests**. That means you need to develop an intuition for when to ignore something, and when to dig in further.
+
+## About you
+
+We're looking for someone with strong front-end skills or someone who has a passion and is eager to learn more in this area. More important than specific technical skills though is that you're a strong problem solver who loves to learn. Details matter to you.
```

Comment:
> We're looking for someone with strong front-end skills

Intern...

Reply:
```suggestion
We're looking for someone with front-end skills or someone who has a passion and is eager to learn more in this area. More important than specific technical skills though is that you're a strong problem solver who loves to learn. Details matter to you.
```

Or maybe just someone passionate for front-end technologies?